apps/ejabberd/src/mod_muc_light.erl

```diff
@@ -373,6 +374,10 @@ can_access_identity(_Acc, _Room, _User) ->
     %% 0045 MUC element with user identity and we don't want it
     false.
 
+-spec delete_room(RoomUS :: ejabberd:simple_bare_jid()) -> ok | {error, not_exists}.
```

Comment:
Shouldn't be a part of "Hook handlers" section. Maybe let's create a new one like "Admin API"?

Reply:
We could create a new section called "Admin API" and add ```try_to_create_room/3``` and ```delete_room/1``` in it. But the bit strange thing is that, the API function ```try_to_create_room/3``` is being called by the internal function ```create_room/5```. So, if it is ok to have this situation of an internal function calling an API function, then it is fine to move them to the new section "Admin API". Whats your opinion Piotr ?

Reply:
I think it's nothing wrong with internal function calling API function.

Reply:
ok Piotr. The code is now updated as per your suggestion.